rita_common/src/blockchain_oracle/mod.rs

```diff
@@ -93,39 +95,32 @@ pub struct BlockchainOracle {
     pub last_updated: Option<Instant>,
 }
 
-/// This struct contains important information to determine when a router should be paying and when it should be enforcing on
-/// other routers
-///
-/// 1.) gas_price : latest gas price value. Note that due to routers taking different times to run a loop, different routers
-///     may have different values for this field. Post EIP1559, the max price change per block is 12.5% of the previous block
-///
-/// 2.) payment_threshold : This is the ammount at which a router will make a payment. Below this value, the router will not may a payment since
-///     a large portion of the payment will be eaten in fees which is not desirable. This is caluculated by
-///     gas_price * gas_required * dynamic_fee_multiplier, where dynamic fee multiplier determines how much larger the payment is
-///     compared to the fees (fees are essentially gas_price * gas_required). When this is larger, the router pays less often and
-///     vice versa
-///
-/// 3.) close_threshold : This is a multiple of payment_threshold and determines how many payments a router can miss before enforcing it.
-///     For ex. if close_thres is 3 * pay_thres, another router may miss upto 3 payments before it gets enforced upon. Another way to think of this
-///     is if a router owes more than the close_thresh, it will get enforced upon.
-#[derive(Debug, Clone)]
-pub struct GasInfo {
-    pub payment_threshold: Int256,
-    pub close_threshold: Int256,
-    pub gas_price: Uint256,
+// Set Xdai default
+fn default_net_version() -> u64 {
+    100u64
 }
 
-fn default_close_threshold() -> Int256 {
-    Int256::from(-1) * default_pay_threshold() * CLOSE_THRESH_MULT.into()
+/// payment_threshold : This is the ammount at which a router will make a payment. Below this value, the router will not may a payment since
+/// a large portion of the payment will be eaten in fees which is not desirable. This is caluculated by
+/// gas_price * gas_required * dynamic_fee_multiplier, where dynamic fee multiplier determines how much larger the payment is
```

Comment:
this comment needs updating as well it discussing the gas price and such.

Reply:
done